lib/index.js

```diff
@@ -118,15 +101,6 @@ function encodeArrayBuffer(packet, supportsBinary, callback) {
  */
 
 exports.encodeBase64Packet = function(packet, callback){
-
-  if (!Buffer.isBuffer(packet.data)) {
-    var buf = new Buffer(packet.data.byteLength);
-    for (var i = 0; i < buf.length; i++) {
-      buf[i] = packet.data[i];
-    }
-    packet.data = buf;
-  }
```

Comment:
Well, ArrayBuffer is converted to Buffer a bit before (there https://github.com/darrachequesne/engine.io-parser/blob/patch-1/lib/index.js#L66):
```javascript
  } else if (packet.data && (packet.data.buffer || packet.data) instanceof ArrayBuffer) {
    >> packet.data = toBuffer(packet.data);
    return encodeBuffer(packet, supportsBinary, callback);
  }
```
So the conversion in `encodeBase64Packet` should not be needed anymore, right?